Model/Sync/Customers/Processor.php

```diff
@@ -321,6 +321,17 @@ public function processEntities($retryCustomerIds = [])
      */
     public function syncCustomer($customer, $realTImeSync = false, $customerAddress = null)
     {
+        $storeId = $this->config->getStoreId();
+        if ($this->config->syncResetInProgress($storeId, 'customer')) {
+            $this->yotpoSmsBumpLogger->info(
+                __(
+                    'Customer sync is skipped because sync reset is in progress - Magento Store ID: %1, Name: %2',
```

Comment:
The same as with the [core PR](https://github.com/YotpoLtd/magento2-module-core/pull/104), why do we put it here?

Reply:
If we don't put it inside this loop, the cron will continue processing items in a batch without considering the sync reset activity.
We will also add the same check at process() method. PR will updated soon